controllers/nstemplateset/space_roles.go

```diff
@@ -109,7 +109,6 @@ func (r *spaceRolesManager) getSpaceRolesObjects(ns *corev1.Namespace, spaceRole
 			objs, err := tierTemplate.process(r.Scheme, map[string]string{
 				Namespace: ns.Name,
 				Username:  username,
-				SpaceName: username,
```

Comment:
You are sure you don't need it here? Just wanted to double check :)

Reply:
It shouldn't be needed because `spacerole_*.yaml` templates from all tiers only use USERNAME parameter. And I saw tests working both locally and in CI.

But you never know until you deploy it to prod  🤦‍♂️